console/command/Test.php

```diff
@@ -67,6 +67,13 @@ class Test extends \lithium\console\Command {
 	public $plain = false;
 
 	/**
+	 * Shows text output with color or without
+	 *
+	 * @var boolean
+	 */
+	public $noColor = false;
```

Comment:
I mentioned that I reverted this, because the `bash console/li3 help test` command does not take into account inherited properties. This is done also with the `plain` flag which already existed.